python/tvm/relay/expr.py

```diff
@@ -113,3 +114,11 @@ class If(Expr):
     def __init__(self, cond, true_value, false_value):
         self.__init_handle_by_constructor__(
             _make.If, cond, true_value, false_value)
+
+_init_api("relay.expr", __name__)
```

Comment:
I like the convention of locating this code in `_expr.py` so we can use `.pyi` files to give type annotations.